tithemi/assignment3/LettersGrid.cpp

```diff
@@ -0,0 +1,71 @@
+#ifndef LETTERSGRID_CPP_
+#define LETTERSGRID_CPP_
+
+#include "LettersGrid.hpp"
+
+static const size_t neighborsCount = 8;
+static const size_t neighborsOffsets[neighborsCount][2] =
+{
+    { -1, -1 },
+    { -1, +0 },
+    { -1, +1 },
+    { +0, -1 },
+    { +0, +1 },
+    { +1, -1 },
+    { +1, +0 },
+    { +1, +1 }
+};
+
+template <size_t Rows, size_t Columns>
+void LettersGrid<Rows, Columns>::findWords(
+    size_t i, size_t j,
+    const Dictionary& dictionary,
+    std::set<std::string>& result,
+    std::string& current,
+    bool visited[Rows][Columns]

```

Comment:
Thanks for splitting the parameters over multiple lines instead of one giant line! Makes the code a lot easier to read.